### PR TITLE
#2: Fix release artifacts' maven coordinates

### PR DESCRIPTION
**Pull request by eparovyshnaya**
- fix `group` and `artifactId`
 - add `LICENSE` file to all the artifacts

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -68,19 +67,24 @@ tasks.getByName("sourcesJar") {
...
 }
 
-fun extendManifest(mf: Manifest): Unit {
+fun extendManifest(mf: Manifest) {
     mf.attributes(
         "Bundle-ManifestVersion" to "2",
-        "Bundle-SymbolicName" to project.name,
-        "Bundle-Name" to project.name,
+        "Bundle-SymbolicName" to "ru.arsysop.lang",
+        "Bundle-Name" to "ru.arsysop.lang",
```
Comment (ruspl-afed) [inline build.gradle.kts]: It should be human readable

File: build.gradle.kts [hunk 2]
```diff
@@ -68,19 +67,24 @@ tasks.getByName("sourcesJar") {
...
         "Bundle-Version" to project.version,
         "Bundle-Vendor" to "ArSysOp",
```
Comment (ruspl-afed) [inline build.gradle.kts:81]: Let's add Bundle-Copyright

**Comment by eparovyshnaya** [comment]
Closes #2

**Review by ruspl-afed — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 2 above)
Comment (eparovyshnaya) [reply build.gradle.kts:81]: Done

File: build.gradle.kts (hunk 1 above)
Comment (eparovyshnaya) [reply build.gradle.kts]: Done

**Review by ruspl-afed — APPROVED** [review]

**Pull request merged**